sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/ConfluentSchemaRegistryDeserializerProvider.java

```diff
@@ -74,17 +75,23 @@
 
   public static <T> ConfluentSchemaRegistryDeserializerProvider<T> of(
       String schemaRegistryUrl, String subject) {
-    return of(schemaRegistryUrl, subject, null);
+    return of(schemaRegistryUrl, subject, null, null);
   }
 
   public static <T> ConfluentSchemaRegistryDeserializerProvider<T> of(
-      String schemaRegistryUrl, String subject, @Nullable Integer version) {
+      String schemaRegistryUrl,
+      String subject,
+      @Nullable Integer version,
+      @Nullable Map<String, ?> schemaRegistryConfigs) {
```

Comment:
Please, add a method with a new signature to keep public API stable.

Reply:
@aromanenko-dev I've added an additional signature, PTAL. thanks!